game.js

```diff
@@ -249,7 +244,7 @@ class Game extends Screen {
 	}
 
 	drawGameOver() {
-		this.drawBackground()
+		this.drawBackground();
```

Comment:
Did you notice I've gone the 'no semi-colon' preference for this project (set in Prettier config).

Reply:
Removed semicolon.